code/datums/diseases/lycancoughy.dm

```diff
@@ -34,12 +35,20 @@
 												"<span class='userdanger'>You howl!</span>")
 				affected_mob.AdjustConfused(rand(6, 8))
 			if(prob(3))
-				var/list/puppytype = list(/mob/living/simple_animal/pet/corgi/puppy, /mob/living/simple_animal/pet/pug, /mob/living/simple_animal/pet/fox)
-				var/mob/living/puppypicked = pick(puppytype)
-				affected_mob.visible_message("<span class='danger'>[affected_mob] coughs up [initial(puppypicked.name)]!</span>", \
-													"<span class='userdanger'>You cough up [initial(puppypicked.name)]?!</span>")
+				if(barklimit <= 10)
+					var/list/puppytype = list(/mob/living/simple_animal/pet/corgi/puppy, /mob/living/simple_animal/pet/pug, /mob/living/simple_animal/pet/fox)
+					var/mob/living/puppypicked = pick(puppytype)
+					affected_mob.visible_message("<span class='danger'>[affected_mob] coughs up [initial(puppypicked.name)]!</span>", \
+														"<span class='userdanger'>You cough up [initial(puppypicked.name)]?!</span>")
+					new puppypicked(affected_mob.loc)
+					new puppypicked(affected_mob.loc)
+					barklimit ++
+				else
+					var/obj/item/toy/plushie/coughfox = /obj/item/toy/plushie/orange_fox
```

Comment:
Why not spawn the corgi plushie as well?

This is still going to be an awful lot of objects created. Yeah, they won't be processing, but it's still going to really pile up horrifically.

Reply:
I mean i could just leave it at the damage if you wanted.